torch/lib/THCUNN/LookupTable.cu

```diff
@@ -1,10 +1,10 @@
 #include "THCUNN.h"
 #include "common.h"
-
 #include "THCThrustAllocator.cuh"
 #include <thrust/device_ptr.h>
 #include <thrust/execution_policy.h>
 #include <thrust/iterator/constant_iterator.h>
+#include <thrust/iterator/zip_iterator.h>
 #include <thrust/transform_reduce.h>
```

Comment:
take a look at the rest of the file and see if it still needs these thrust headers